I/IGLWrap/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+#=
+BINARYBUILDER_RUNNER=privileged julia build_tarballs.jl --verbose --debug       
+=#
+
+using BinaryBuilder
+using Pkg
+
+version = v"2.3.0" # libIGL version, see below:
+
+sources = [
+	DirectorySource("./iglwrap"),
+	# 2.3.0 stable release:
+	GitSource("https://github.com/libigl/libigl.git",
+		"e60423e28c86b6aa2a3f6eb0112e8fd881f96777")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir -p build
+cd build
+# this line seems to prevent compilation; removed for testing:
+# -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
```

Comment:
I presume you're going to reinstate this line later?  I mean, this is what makes cross-compilation works correctly with CMake, otherwise it may pick up wrong tools for the wrong platform.

If you're happy to support only x86_64-linux-gnu that may be ok, but for most of the other platforms this won't work at all.  And for the record, cross-compilation was failing because they do a try_run for [what is mostly a compiler check](https://github.com/CGAL/cgal/blob/5759a800e35e36fb09666c5d5f94f6479b778d63/Installation/cmake/modules/config/support/CGAL_test_cpp_version.cpp)

Reply:
I'm trying to set Cmake variables to [tell CGAL to behave](https://github.com/CGAL/cgal/blob/master/Installation/cmake/modules/CGAL_Macros.cmake), we'll see if that works

Reply:
I [opened an issue on CGAL](https://github.com/CGAL/cgal/issues/6101) to ask how to get the cross-compilation working.
If that issue gets ignored, I would be happy with a Linux64-only package (at least at first) — as this is the github platform anyway, this allows me to e.g. host some `Documenter.jl` depending on this package.

Reply:
Note that libIGL is under GPL, and not MPL. At least its dependency CGAL is under GPLv3+.

Note also that libIGL CMake scripts silently download CGAL version 4.12, that is pretty old (2018). More recent versions of CGAL are header-only, and "behave" a lot better. In recent versions of CGAL, CMake is also useless (unless downstream projects use CMake as well).


Jérôme, that old piece of code in `CGAL_Macros.cmake` is one of my own, but it was at the time CMake was version 2.6, and C++11 was still called C++0x. Nowdays, we no longer use those lines of CMake scripts. We still have to clean up a bit our files, to remove all the unused old CMake code.